src/components/Layout/WelcomeHero/index.js

```diff
@@ -15,9 +15,9 @@ function WelcomeHero({ title, description }) {
         <div className={styles.cta}>
           <Link
             className={clsx("button button--primary button--lg", styles.button)}
-            to="/discover-cardano"
+            to="/where-to-get-ada"
           >
-            Discover Cardano
+            Where to get Ada?
```

Comment:
Ada should be written in lower case. When talking about the cryptocurrency, do not capitalize, unless at the beginning of a sentence. The idea behind this is to treat it like dollars or euros.